src/main/java/racingcar/domain/Participants.java

```diff
@@ -0,0 +1,31 @@
+package racingcar.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.NoSuchElementException;
+
+public class Participants {
+    private final List<Car> participants;
+
+    public Participants(List<Car> cars) {
+        this.participants = cars;
+    }
+
+    public List<Car> getParticipants() {
+        return Collections.unmodifiableList(participants);
+    }
+
+    public List<Car> getWinners() {
+        Integer leadingPosition = getLeadingPosition();
+        return participants.stream()
+                .filter(car -> leadingPosition.equals(car.getPosition()))
+                .toList();
+    }
+
+    private Integer getLeadingPosition() {
+        return participants.stream()
+                .mapToInt(Car::getPosition)
+                .max()
+                .orElseThrow(NoSuchElementException::new);
```

Comment:
위치의 기본값을 `0`으로 주었으니, 아래 코드와 같이 수정하는 건 어떨까요?
```suggestion
                .orElse(0);
```